src/index.html

```diff
@@ -1,17 +1,217 @@
+            id="termsOfAgreement"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="forms">
+        <legend>Registration</legend>
+        <div class="formField">
+          <label for="email">
+            E-mail:
+          </label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="formField">
+          <label for="password">
+            Password:
+          </label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            maxlength="16"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="forms">
+        <legend>An interesting fact about you!</legend>
+        <div class="formField">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats-lover"
+            id="cats-lover1"
+            value="yes"
+          >
+          <label for="cats-lover1">
+            Yes
+          </label>
+          <input
+            type="radio"
+            name="cats-lover"
+            id="cats-lover2"
+            value="yes"
+          >
+          <label for="cats-lover2">
+            No
+          </label>
+        </div>
+
+        <div class="formField">
+          <label for="color">
+            What is your favorite color?
+          </label>
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </div>
+
+        <div class="formField">
+          <label for="goToBed">
+            What time do you go to bed?
+          </label>
+          <input
+            type="time"
+            name="goToBed"
+            id="goToBed"
+          >
+        </div>
+
+        <div class="formField">
+          <label for="favoriteСar">
+            What are your favorite brand of cars?
+          </label>
+          <select
+            name="favoriteСar"
+            id="favoriteСar"
+            multiple=""
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+
+        <div class="formField">
+          <label for="rating">
+            How do you rate our work?
+          </label>
+          <input
+            type="range"
+            name="rating"
+            id="rating"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="forms">
+        <legend> Additional info: </legend>
+        <div class="formField">
+          <label for="comments">
+            Comments:
+          </label>
+          <textarea name="comments" id="comments"></textarea>
+        </div>
+
+        <div class="formField">
+          <label for="recom">
```

Comment:
Do not shorten your namings, it's a bad practice
```suggestion
          <label for="recommmend">
```